src/main/java/edu/ie3/datamodel/io/FileNamingStrategy.java

```diff
@@ -205,6 +205,15 @@ private static String prepareSuffix(String suffix) {
     return Optional.of(buildResultEntityString(resultEntityClass));
   }
 
+  /**
+   * Get the the file name for coordinates
+   *
+   * @return the filename string
+   */
+  public Optional<String> getCoordinateFileName() {
```

Comment:
Why do you return an `Optional` here? This Optional is never empty

Reply:
I wanted to keep the structure the others had